src/Application/Actions/Hooks/StripePaymentsUpdate.php

```diff
@@ -458,15 +465,36 @@ private function doPostMarkRefundedUpdates(Donation $donation, bool $isCoreDonat
 
     private function handleCashBalanceUpdate(Event $event, Response $response): Response
     {
-        if (getenv('APP_ENV') === 'production') {
+        Assertion::eq('customer_cash_balance_transaction.created', $event->type);
+
+        /** @var array{customer: string, currency: string, net_amount: int, ending_balance: int, type: string} $webhookObject */
+        $webhookObject = $event->data->toArray()['object'];
+
+        $stripeAccountId = StripeCustomerId::of($webhookObject['customer']);
+
+        $this->chatter->send(new ChatMessage(
```

Comment:
Should we think about switching to an info log for the "happy path", so we can check what notifications we sent out on-demand if there are queries, while not having the noise of a Slack alert on every top up?

Reply:
I'd like to have some slack alerts for the first few times it happens, but then yes it would make sense to delete it. I'll add the info log now, if you think we should delete the send in the happy path I'll delete that as well.

Although this probably should be moved underneath the check for whether the type is 'funded' - otherwise this would give us alerts on bottom downs as well as top ups.

Reply:
Actually done in 6977c8b.